d3a_api_client/utils.py

```diff
@@ -264,3 +264,25 @@ def log_bid_offer_confirmation(message):
                          f"{round(energy, 2)} kWh at {price} cts/kWh")
     except Exception as e:
         logging.error(f"Logging bid/offer info failed.{e}")
+
+
+def flatten_info_dict(indict: dict) -> dict:
```

Comment:
Another optional idea: the flattened data can only be obtained by using a function on the grid_tree data in this implementation. Another approach could be, to save the last market info and tick_info in the Client class. That way the user could automatically access the market_info using `self.market_info` and `self.flat_market_info` , same for tick_info too.

Reply:
I like the latter a lot and it is straight forward too. Will add it.